packages/location-management/src/components/LocationTagView/index.tsx

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable @typescript-eslint/camelcase */
+import React, { useEffect, useState } from 'react';
+import { Helmet } from 'react-helmet';
+import { Row, Col, Menu, Dropdown, Button, Divider, Input, notification } from 'antd';
+import { SettingOutlined, PlusOutlined } from '@ant-design/icons';
+import LocationTagDetail, { LocationTagDetailProps } from '../LocationTagDetail';
+import { SearchOutlined } from '@ant-design/icons';
+import { useDispatch, useSelector } from 'react-redux';
+import { OpenSRPService } from '@opensrp/server-service';
+import reducerRegistry from '@onaio/redux-reducer-registry';
+import reducer, {
+  fetchLocationTags,
+  getLocationTagsArray,
+  LocationTag,
+  reducerName,
+} from '../../ducks/location-tags';
+import { getAccessToken } from '@onaio/session-reducer';
+import { API_BASE_URL, LOCATION_TAG_ALL, URL_LOCATION_TAG_ADD } from '../../constants';
+import Table, { TableData } from './Table';
+import './LocationTagView.css';
+import { Ripple } from '@onaio/loaders';
+import { Link } from 'react-router-dom';
+
+reducerRegistry.register(reducerName, reducer);
+
+const LocationTagView: React.FC = () => {
+  const accessToken = useSelector((state) => getAccessToken(state) as string);
+  const locationsArray = useSelector((state) => getLocationTagsArray(state));
+  const dispatch = useDispatch();
+  const [detail, setDetail] = useState<LocationTagDetailProps | null>(null);
+  const [isLoading, setIsLoading] = useState<boolean>(true);
+  const [value, setValue] = useState('');
+  const [filter, setfilterData] = useState<TableData[] | null>(null);
+
+  useEffect(() => {
+    if (isLoading) {
+      const serve = new OpenSRPService(accessToken, API_BASE_URL, LOCATION_TAG_ALL);
+      serve
+        .list({ is_jurisdiction: true, serverVersion: 0 })
+        .then((response: LocationTag[]) => {
+          dispatch(fetchLocationTags(response));
+          setIsLoading(false);
+        })
+        .catch((e) => notification.error({ message: `${e}`, description: '' }));
+    }
+  });
+
+  const tableData: TableData[] = [];
+
+  if (locationsArray.length) {
+    locationsArray.forEach((location: LocationTag, i: number) => {
+      tableData.push({
+        key: i.toString(),
+        id: location.id,
+        name: location.name,
+        active: location.active,
+        description: location.description,
+      });
+    });
+  }
+
+  const onChange = (e: { target: { value: string } }) => {
```

Comment:
is it possible to get a more accurate type for the event parameter,

Reply:
done